platforms/python/prereqs/build.sh

```diff
@@ -31,8 +33,6 @@ PYTHON_GET_PIP_URL="https://github.com/pypa/get-pip/raw/3cb8888cc2869620f57d5d2d
         libbluetooth-dev \
         tk-dev \
         uuid-dev \
-        # Adding additional python packages to support all optional python modules:
-        # https://devguide.python.org/getting-started/setup-building/index.html#install-dependencies
```

Comment:
do you know why this failed, while https://github.com/microsoft/Oryx/blob/main/images/build/Dockerfiles/gitHubActions.Dockerfile#L6-L52 does not cause issues?

Reply:
Yes, you are referencing a Dockerfile. So `#` comments: https://docs.docker.com/engine/reference/builder/
Are ignored during docker build process.